modules/xmpp/ChatRoom.js

```diff
@@ -148,6 +148,16 @@ export default class ChatRoom extends Listenable {
         // Jicofo makes decisions based on that. Initialize it with 'false'
         // here.
         this.addVideoInfoToPresence(false);
+
+        if (options.deploymentInfo && options.deploymentInfo.userRegion) {
+            this.presMap.nodes.push(
+                { 'tagName': 'region',
```

Comment:
What you said sounds correct. Normally when passing in a multi-attribute object it's something like:
```javascript
functionName({
  key1: val1,
  key2: val2
});
```